packages/office-addin-lint/package.json

```diff
@@ -25,9 +25,9 @@
     "@typescript-eslint/eslint-plugin": "^2.26.0",
```

Comment:
Are there newer versions of the typescript-eslint packages as well?

Should we also update the prettier version?

Reply:
Yes there are, they just don't need to be updated for the sake of fixing this ```lodash``` vulnerability bug.
I will update them for the sake of completeness.